Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.nuspec

```diff
@@ -6,9 +6,17 @@
     <title>Windows Community Toolkit Notifications</title>
     <authors>Microsoft.Toolkit,dotnetfoundation</authors>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</licenseUrl>
-    <projectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</iconUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+<package>
+  <metadata>
+    <icon>images\nuget.png</icon>
```

Comment:
we're already in the package/metadata tag here (look above), so we don't have to duplicate it.

These can just go alongside the other tags here.

Again, we should also keep the `iconUrl` one from before based [on the doc here](https://docs.microsoft.com/en-us/nuget/reference/nuspec#icon).